component/src/test/java/org/wso2/extension/siddhi/execution/stats/MedianAttributeAggregatorTestCase.java

```diff
@@ -303,5 +305,274 @@ public void receive(org.wso2.siddhi.core.event.Event[] events) {
+                            break;
+                    }
+                }
+            }
+        });
+
+        InputHandler inputHandler = siddhiAppRuntime.getInputHandler("inputStream");
+        siddhiAppRuntime.start();
+        inputHandler.send(new Object[]{1.5f});
+        inputHandler.send(new Object[]{2.5f});
+        inputHandler.send(new Object[]{3.5f});
+        inputHandler.send(new Object[]{4.5f});
+        inputHandler.send(new Object[]{5.5f});
+        inputHandler.send(new Object[]{6.5f});
+        inputHandler.send(new Object[]{10.5f});
+        inputHandler.send(new Object[]{9.5f});
+        inputHandler.send(new Object[]{8.5f});
+        inputHandler.send(new Object[]{7.5f});
+
+        siddhiAppRuntime.shutdown();
+    }
+
+    @Test
+    public void test6() throws InterruptedException {
+
+        log.info("MedianAggregatorTestCase Double Sliding Length Window TestCase");
+        SiddhiManager siddhiManager = new SiddhiManager();
+
+
+        String inStreamDefinition = "define stream inputStream (tt float); " +
+                "define stream outputStream (tt double);";
+
+        String query = "@info(name = 'query1') " + "from inputStream#window.lengthBatch(5) " +
+                "select stats:median(tt) as tt insert into filteredOutputStream";
+        SiddhiAppRuntime siddhiAppRuntime = siddhiManager.
+                createSiddhiAppRuntime(inStreamDefinition + query);
+
+        siddhiAppRuntime.addCallback("filteredOutputStream", new StreamCallback() {
+            @Override
+            public void receive(org.wso2.siddhi.core.event.Event[] events) {
+                for (Event ev : events) {
+                    count++;
+                    switch (count) {
+                        case 1:
+                            AssertJUnit.assertEquals(3.5, ev.getData()[0]);
+                            break;
+                        case 2:
+                            AssertJUnit.assertEquals(8.5, ev.getData(0));
+                            break;
+                    }
+                }
+            }
+        });
+
+        InputHandler inputHandler = siddhiAppRuntime.getInputHandler("inputStream");
+        siddhiAppRuntime.start();
+        inputHandler.send(new Object[]{1.5f});
+        inputHandler.send(new Object[]{2.5f});
+        inputHandler.send(new Object[]{3.5f});
+        inputHandler.send(new Object[]{4.5f});
+        inputHandler.send(new Object[]{5.5f});
+        inputHandler.send(new Object[]{6.5f});
+        inputHandler.send(new Object[]{10.5f});
+        inputHandler.send(new Object[]{9.5f});
+        inputHandler.send(new Object[]{8.5f});
+        inputHandler.send(new Object[]{7.5f});
+
+        siddhiAppRuntime.shutdown();
+    }
+
+    @Test
+    public void test7() throws InterruptedException {
+
+        log.info("MedianAggregatorTestCase Double Sliding Length Window TestCase");
+        SiddhiManager siddhiManager = new SiddhiManager();
+
+
```

Comment:
Remove unnecessary line spaces.

Reply:
Do the same for other test cases as well.

Reply:
fixed in 748f4d3